subprojects/packagefiles/libcap/generator.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+exec grep -E '^#define\s+CAP_[A-Z0-9_]+\s+[0-9]+\s*$' "$1" | \
```

Comment:
Nit: in a pipeline, `exec` doesn't do what it looks like it's doing, and it isn't really necessary here, so I'd drop.